src/modules/tests/pre-test-declarations/cat-a-mod1/__tests__/pre-test-declarations.cat-a-mod1.reducer.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { preTestDeclarationsCatAMod1Reducer, initialState } from '../pre-test-declarations.cat-a-mod1.reducer';
-import { CbtNumberChanged } from '../pre-test-declarations.cat-a-mod1.actions';
+import { CbtNumberChanged } from '../../cat-a/pre-test-declarations.cat-a.actions';
```

Comment:
why is the directory now just called `cat-a` as opposed to `cat-a-mod1`

Reply:
Are we just not splitting out the actions for `cat-a` categories?

Reply:
Mod 1 and Mod 2 both use the same cbt number so instead of duplicating this file twice I've just moved the actions file in to a cat a folder, which they both use :)

Reply:
Perfect, thanks for the explanation. :)

Reply:
no problem :)